<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C5SaleFromStoreOrder</class>
 <widget class="QDialog" name="C5SaleFromStoreOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sale</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Partner</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="C5LineEdit" name="teTime">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="4">
       <widget class="C5LineEditWithSelector" name="lePartnerName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="4">
       <widget class="C5LineEditWithSelector" name="leSellerName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Order id</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QCheckBox" name="chDebt">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Debt</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>376</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Seller</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="C5LineEdit" name="leUserId">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="C5LineEditWithSelector" name="leSeller"/>
      </item>
      <item row="2" column="8">
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="C5DateEdit" name="deDate">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="C5LineEdit" name="leID">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="C5LineEditWithSelector" name="lePartner"/>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>UUID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="C5LineEditWithSelector" name="leUUID">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="btnCopyUUID">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/copy.png</normaloff>:/copy.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="C5TableWidget" name="tblData">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Qty</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Store</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TaxDept</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Adgt</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="1">
       <widget class="C5LineEdit" name="leTotalCard">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="C5LineEdit" name="leTotal">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Amount, cash</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="C5LineEdit" name="leTotalCash">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Amount, total</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="btnPrintTax">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Print tax</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Amount, card</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>513</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5TableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">c5tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
